src/style.css

```diff
@@ -1,3 +1,113 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+
 }
+
+.header {
+  background: #fff;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: relative;
+}
+
+@media (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+
+}
+@media (min-width: 1200px) {
+  .header {
+    width: 100%;
+  }
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 50px;
+}
+
+.logo-moyo {
+  display: flex;
+}
+
+.nav {
+  position: absolute;
+  right: 50px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+
+}
+
+.nav__item {
+  line-height: 60px;
+  margin-left: 20px;
+  list-style: none;
+  text-transform: uppercase;
+  height: 60px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.active {
+  text-decoration: none;
+  color: #000;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  font-size: 12px;
+  font-weight: 500;
+  position: relative;
+}
+
+.active:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  display: block;
+  background: #00acdc;
+  content: " ";
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+  width: 100%;
+  bottom: 0;
+}
+
+/* This comment for activation :target::after from all links */
```

Comment:
Unused code should be removed. If you need to preserve it for future reference, consider maintaining a separate branch or documenting it outside the production codebase.

Reply:
Done